tests/labhub_test.py

```diff
@@ -142,6 +152,9 @@ def test_create_issue_cmd(self):
         self.mock_team.is_member.return_value = True
         testbot_public = self
 
+        mock_nick = create_autospec(Message)
+        mock_nick.frm.nick.return_value = True
+
```

Comment:
Creating these `mock_nick`s aren't doing any good, because you are not passing the mocked `Message` object anywhere.

Reply:
Then how do I do this? The tests system is not comfortable with changes.